app/views/pages/about.html.erb

```diff
@@ -84,7 +83,7 @@
     <h2>Supported Package Managers</h2>
     <hr>
     <p>
-      Libraries.io supports the following package managers. For more detail check out <a href='<%= url_for(controller: 'pages', action: 'pmmatrix',) %>'>our compatibility matrix</a>.
+      Libraries.io supports the following package managers. For more detail check out <a href='<%= compatibility_path %>'>our compatibility matrix</a>.
```

Comment:
Using the named route helpers means that if we decide to move the action to a different controller, we won't need to go through the whole application and update every link pointing it.